### DISCUSSION TITLE
Hubs-cloud personal failing to create stack

### DISCUSSION THREAD

**Discussion by lukewhitingdev** [mozilla/hubs#2857] [Hubs Cloud Questions] [answered]
Hi, been trying to deploy a custom hubs cloud personal stack on a aws ec2 for a while but i run into the same failure problem.
Info:
Domains are registered on route53 (both .link and .co.uk)
I have read and followed the instructions on the hubs docs pages etc.
I have already tried disabling transfer lock's on my domains
- Side note there is already a hubs cloud stack running on the same server (will that cause a issue?)

Any help is appreciated :)

![123](https://user-images.githubusercontent.com/43708922/90157207-5c785400-dd85-11ea-8d7b-8df469ef936d.JPG)

**Comment by ocularvr** [comment, marked as answer]
Follow your log along to where the errors start. Usually it is in the KEY PAIRS section - you need at least one.
Send me a screen capture of where the errors start and we can be sure why

**Reply by lukewhitingdev** [reply]
okay this seems to be the offending item, any idea how id get to the log that its referring to? i looked in cloudwatch but cant figure out how to find it. 
![12312312](https://user-images.githubusercontent.com/43708922/90158575-115f4080-dd87-11ea-806b-eeb171aca836.JPG)

**Reply by lukewhitingdev** [reply]
also seems to be my domain that ends in .co.uk is there specific problems with that TLD? since id rather not buy a new domain to fix the problem :(

**Reply by ocularvr** [reply]
Have a look at this:
https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/hosted-zones-private.html
I think your issue is the fact that you transferred those two domains over to aws.
If im wrong then best bet would be to open a ticket.
I think its your Route 53 setup friend

**Reply by lukewhitingdev** [reply]
hmm that's funky, i registered the domains directly through route 53 but yeah ill open a ticket and maybe get a new domain and try that to see if that fixes it. Cheers for the help :D
